src/DQNModel.py

```diff
@@ -335,10 +335,11 @@ def _calculate_loss(self, transitions, discount_factor):
         curr_state = torch.tensor(transitions[0])
         next_state = torch.tensor(transitions[3])
         terminal = torch.tensor(transitions[4]).type(torch.int)
-
+        
```

Comment:
Nitpicking but could you remove the changes in this file if it's just white spaces?